[2025-10-11T06:04:32.150+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-11T06:04:32.212+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipeline_flow.check_create_stg_tables scheduled__1998-05-03T00:00:00+00:00 [queued]>
[2025-10-11T06:04:32.229+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipeline_flow.check_create_stg_tables scheduled__1998-05-03T00:00:00+00:00 [queued]>
[2025-10-11T06:04:32.231+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-10-11T06:04:32.259+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): check_create_stg_tables> on 1998-05-03 00:00:00+00:00
[2025-10-11T06:04:32.282+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=9119) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-10-11T06:04:32.284+0000] {standard_task_runner.py:64} INFO - Started process 9132 to run task
[2025-10-11T06:04:32.283+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'pipeline_flow', 'check_create_stg_tables', 'scheduled__1998-05-03T00:00:00+00:00', '--job-id', '376', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_dag.py', '--cfg-path', '/tmp/tmps5e66sk6']
[2025-10-11T06:04:32.289+0000] {standard_task_runner.py:91} INFO - Job 376: Subtask check_create_stg_tables
[2025-10-11T06:04:32.389+0000] {task_command.py:426} INFO - Running <TaskInstance: pipeline_flow.check_create_stg_tables scheduled__1998-05-03T00:00:00+00:00 [running]> on host 321597a50810
[2025-10-11T06:04:32.550+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='oluwaseyiogundimu10@gmail.com' AIRFLOW_CTX_DAG_OWNER='seyi' AIRFLOW_CTX_DAG_ID='pipeline_flow' AIRFLOW_CTX_TASK_ID='check_create_stg_tables' AIRFLOW_CTX_EXECUTION_DATE='1998-05-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__1998-05-03T00:00:00+00:00'
[2025-10-11T06:04:32.552+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-11T06:04:32.579+0000] {connection.py:22} INFO - Successfully connected to Postgres.
[2025-10-11T06:04:32.681+0000] {connection.py:43} INFO - Table 'stg_users' in Postgres.
[2025-10-11T06:04:32.683+0000] {connection.py:22} INFO - Successfully connected to Postgres.
[2025-10-11T06:04:32.777+0000] {connection.py:43} INFO - Table 'users' in Postgres.
[2025-10-11T06:04:32.779+0000] {connection.py:22} INFO - Successfully connected to Postgres.
[2025-10-11T06:04:32.876+0000] {connection.py:43} INFO - Table 'stg_movies' in Postgres.
[2025-10-11T06:04:32.877+0000] {connection.py:22} INFO - Successfully connected to Postgres.
[2025-10-11T06:04:32.971+0000] {connection.py:43} INFO - Table 'movies' in Postgres.
[2025-10-11T06:04:32.973+0000] {connection.py:22} INFO - Successfully connected to Postgres.
[2025-10-11T06:04:33.068+0000] {connection.py:43} INFO - Table 'stg_ratings' in Postgres.
[2025-10-11T06:04:33.070+0000] {connection.py:22} INFO - Successfully connected to Postgres.
[2025-10-11T06:04:33.170+0000] {connection.py:43} INFO - Table 'ratings' in Postgres.
[2025-10-11T06:04:33.171+0000] {load.py:32} INFO - Tables exist.
[2025-10-11T06:04:33.175+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-10-11T06:04:33.178+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-11T06:04:33.200+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=pipeline_flow, task_id=check_create_stg_tables, run_id=scheduled__1998-05-03T00:00:00+00:00, execution_date=19980503T000000, start_date=20251011T060432, end_date=20251011T060433
[2025-10-11T06:04:33.265+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-10-11T06:04:33.314+0000] {taskinstance.py:3503} INFO - 3 downstream tasks scheduled from follow-on schedule check
[2025-10-11T06:04:33.318+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
